KarhooUISDK/UIComponents/LegalNotice/KarhooLegalNoticeView.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  KarhooLegalNoticeView.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 26/01/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public struct KHLegalNoticeViewID {
+    public static let view = "legal_notice_view"
+    public static let button = "legal_notice_button"
+    public static let text = "legal_notice_text"
+}
+
+final class KarhooLegalNoticeView: UIView, UITextViewDelegate {
+    
+    let shouldBeDisplayed: Bool = UITexts.Booking.legalNoticeText.isNotEmpty
+    private weak var parentViewController: UIViewController?
+    private let legalNoticeMailComposer: KarhooLegalNoticeMailComposerProtocol
+    private var zeroHeightTextConstreint: NSLayoutConstraint
+    
+    private var didSetUpConstraints: Bool = false
+    private var legalNoticeTextView: UITextView!
+    
+        private lazy var legalNoticeButton: KarhooExpandViewButton = {
+            let button = KarhooExpandViewButton(title: UITexts.Booking.legalNotice, onExpand: hideLegalNoticePressed, onCollapce: showLegalNoticePressed)
+            button.accessibilityIdentifier = KHLegalNoticeViewID.button
+            button.anchor(height: 44.0)
+            return button
+        }()
+    
+    private var attributedLabel: UILabel = {
```

Comment:
This view reference does not change, so change to `lazy var` or `let`

Reply:
As long as it reference to self it expect to be lazy to work correctly.